packages/sdk/odahuflow/sdk/clients/model.py

```diff
@@ -87,13 +96,23 @@ def __init__(self, url=None, token=None, http_client=requests,
         :type http_exception: python class that implements Exception class interface
         :param timeout: timeout for connections
         :type timeout: int
+        :param client_id: client_id for Client Credentials OAuth2 flow
+        :type client_id: str
+        :param client_secret: client_secret for Client Credentials OAuth2 flow
+        :type client_secret: str
+        :param issuer_url: url for credential login
+        :type issuer_url: str
         """
         self._url = url
-        self._token = token
         self._http_client = http_client
         self._http_exception = http_exception
         self._timeout = timeout
 
+        token = token if token else odahuflow.sdk.config.API_TOKEN
+        issuer_url = issuer_url if issuer_url else odahuflow.sdk.config.ISSUER_URL
+
+        self._authenticator = Authenticator(client_id, client_secret, True, url, token, issuer_url)
```

Comment:
I'd prefer named arguments here, because this `True` seems confusing without any indication what it means.